app/views/teacher_training_adviser/steps/_subject_not_found.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="govuk-heading-l">Get support</h1>
+
+<p>Our teacher training advisers are here to support those returning to teach maths, physics or languages.</p>
+<p>
```

Comment:
This line is really long in the screenshot. It _might_ be worth wrapping the section in a `.govuk-grid-column-two-thirds`